src/actions/authActions/socialAuth.js

```diff
@@ -0,0 +1,34 @@
+import { SOCIAL_AUTH, LOGIN } from "../actionTypes";
+import axios from "axios";
+
+const SocialLogin = payload=>({
+  type:SOCIAL_AUTH,
+  payload
+});
+const LogIn = payload=>({
+  type:LOGIN,
+  payload
+});
+
+export const facebookAuth  =  (acessToken)=>  dispatch =>{
+   
+  axios.post("https://ah-backend-lannister-staging.herokuapp.com/rest-auth/facebook/", acessToken)
+    .then(res => {
+      localStorage.setItem("token",res.data.user.token);
+      dispatch(SocialLogin(res.data.user));
+
+      dispatch(LogIn(true));
+
+    });};
+export const googleAuth = (acessToken)=> dispatch =>{
+   
+  axios.post("https://ah-backend-lannister-staging.herokuapp.com/rest-auth/google/", acessToken)
+   
```

Comment:
Add a base url variable and call it wherever its needed so that you don't repeat the url i.e BASE_URL="https://ah-backend-lannister-staging.herokuapp.com/"

Reply:
noted